### ISSUE TITLE
inconsistent local/global assignment in the documentation library

### ISSUE THREAD

**Issue by u-fischer** [T-F-S/tcolorbox#144]
This here

~~~~
\RequirePackage[check-declarations,enable-debug]{expl3}
\documentclass{article}
\usepackage{tcolorbox}
\tcbuselibrary{documentation}
\begin{document}
\begin{docCommand}{tagpdfsetup}{\marg{key-val-list}}\end{docCommand}

\end{document}
~~~~

errors with 

~~~~
! LaTeX3 Error: Inconsistent local/global assignment

For immediate help type H <return>.
 ...                                              
                                                  
l.10 ...Command}{tagpdfsetup}{\marg{key-val-list}}
                                                  \end{docCommand}
? h

This is a coding error.

Local assignment to a global variable '\g__tcobox_label_seq'.

? x
~~~~

as the library uses in two places `\seq_clear:N` instead of `\seq_gclear:N` .

**Comment by T-F-S** [comment]
Thank you for reporting this. Also for hinting me to `\RequirePackage[check-declarations,enable-debug]{expl3}` for checking.

I will correct this for the next version. Am I right that this bug can be considered as of low priority? Or are there reasons to publish a corrected version fast?

**Comment by u-fischer** [comment]
It is low priority. It only hits people like me which enable `\RequirePackage[check-declarations,enable-debug]{expl3}` to check their own code and then get more errors than expected. ;-)

**Comment by T-F-S** [comment]
I'm just checking `csvsimple-l3.sty` and correct similar errors.

What I cannot correct there are errors about undeclared token lists like `\csvcoli`, `\csvcolii` which are user macros and cannot be declared in advance. Do I (and you) have to live with such errors when `\RequirePackage[check-declarations,enable-debug]{expl3}` is enabled or is there a recommended way to avoid these errors?

**Comment by T-F-S** [comment]
Hopefully fixed with `5.0.0 (2021-12-16)`.
